snakemake/jobs.py

```diff
@@ -1461,7 +1461,9 @@ def cleanup(self):
             job.cleanup()
 
     def postprocess(self, error=False, **kwargs):
-        for job in self.jobs:
+        # Iterate over jobs in toposorted order (see self.__iter__) to
+        # ensure that outputs are touched in correct order.
+        for job in self:
             job.postprocess(error=error, **kwargs)
```

Comment:
```suggestion
        for level in self.toposorted:
            for job in level:
                # postprocessing involves touching output files (to ensure that
                # modification times are always correct. This has to happen in
                # topological order, such that they are not mixed up.
                job.postprocess(error=error, **kwargs)
```

Reply:
Sure, this looks good and it's easier to understand the meaning.